File: 2026-09/src/day_12/prms_181874/Solution.java
package day_12.prms_181874;

class Solution {
    public String solution(String myString) {
        StringBuilder sb = new StringBuilder();
        for (char c : myString.toCharArray()) {
            if (c == 'a') {
                sb.append('A');
            } else if ('B' <= c && c <= 'Z') {
                sb.append(Character.toLowerCase(c));
            } else {
                sb.append(c);
            }
        }

        return sb.toString();
    }
}

File: 2026-09/src/day_12/prms_181874/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

string solution(string myString) {
    string s;
    for (char c : myString) {
        if (c == 'a') {
            s += 'A';
        } else if ('B' <= c && c <= 'Z') {
            s += tolower(c);
        } else {
            s += c;
        }
    }

    return s;
}
